UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Startup.cs

```diff
@@ -36,9 +37,12 @@ public Startup(IWebHostEnvironment env)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers()
-                .AddNewtonsoftJson();
+            services.AddControllers().AddNewtonsoftJson();
             services.Configure<EventHubLoggingConfiguration>(_configuration.GetSection("EventHubLoggingConfiguration"));
+            services.Configure<EnsConfiguration>(_configuration.GetSection("EnsConfiguration"));
+            services.Configure<D365PayloadKeyConfiguration>(_configuration.GetSection("D365PayloadKeyConfiguration"));
+            services.Configure<EnsConfiguration>(_configuration.GetSection("EnsConfiguration"));
+
```

Comment:
Remove duplicate entry for services.Configure<EnsConfiguration>(_configuration.GetSection("EnsConfiguration"));

Reply:
Removed duplicate entry.